libcontainer/mount_linux.go

```diff
@@ -121,3 +179,75 @@ func syscallMode(i fs.FileMode) (o uint32) {
 	// No mapping for Go's ModeTemporary (plan9 only).
 	return
 }
+
+// mountFd creates an open_tree(2)-like mount fd from the provided
```

Comment:
This function is not only used for id mapped mount. So we should change this description to more clearly.

Reply:
I don't think it implies that it's only used for id-mapped mounts ("open_tree(2)-like mount fd from the provided configuration" sounds like it's a mount that is being configured and is a file descriptor representing a mount side a-la `open_tree(2)` but not necessary using `open_tree(2)`). But I can add a note that it is used for other kinds of mounts too.